SonarQube.Common/ProcessRunner.cs

```diff
@@ -148,7 +148,7 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs e)
         {
             if (e.Data != null)
             {
-                this.outputLogger.LogDebug(e.Data);
+                this.outputLogger.LogInfo(e.Data);
```

Comment:
It's important to keep this at INFO level since this logging call redirects logging for child processes, i.e. from the sonar-runner. I'll add a comment about this.